src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpApplicationController.cs

```diff
@@ -103,46 +103,62 @@ public async Task<IActionResult> Applications()
             var applications = await _apiClient.GetApplications(signinId, false);
             applications = applications.Where(app => app.ApplicationStatus != ApplicationStatus.Rejected).ToList();
```

Comment:
Do we know if there's a potential null Reference issue here?

Reply:
GetApplications returns a list of Apply objects, so I don't think there's an issue (also, this is unchanged code).